### ISSUE TITLE
license says MIT at first but it's a BSD style licese

### ISSUE THREAD

**Issue by qlcom** [xxxserxxx/gotop#147]
fix license

**Comment by xxxserxxx** [comment]
Thanks for recording this; it's been a discussion in the chat room that hasn't been well documented.

Issue #36 declares the intention to switch to MIT; contributors accept the switch to MIT; the license itself declares that it's MIT.  Wherever I grabbed the license when I switched, it had an MIT label but clearly had BSD wording.  

The MIT license is regarded as functionally equivalent to the BSD 2-clause license, and variants commonly add wording about using authors and contributors in advertising, which makes it equivalent to BSD-3.  However, since the contributors were asked to approve the MIT license, it's easier to pick BSD-3-like MIT license variant than ask everybody to re-approve a BSD license.

Therefore, to avoid further confusion on this matter, I'm replacing the BSD-ish text in the current licenses with the [Festival variant of the MIT license](https://fedoraproject.org/wiki/Licensing:MIT?rd=Licensing/MIT#Festival_variant).  While it's slightly more restrictive than the BSD-3 wording that I originally chose in the switch, it is a recognized MIT variant and includes the advertising clause.  I'm also including a URL pointing to the source of the variant.

This may not solve the issue that github auto-detects the licenses using a library that may, or may not, correctly recognize the license, but at least the reasoning will be documented in this issue for future posterity.

This *should* be a legal NOP that preserves the original intention of choosing an MIT license.
